data/ita/Trovato_2004_Sannazaro-Arcadia/stemma.gv

```diff
@@ -0,0 +1,60 @@
+digraph {
+Na[label="1504Na"]
+Ald[label="1514 Ald"]
+Rusc[label="1515 Rusc"]
+Zopp1[label="1521 Zopp"]
+Zopp2[label="1524 Zopp"]
+Pag[label="1515 Pag"]
+Fi1[label="1515 Fi"]
+Tac[label="1519 Tac"]
+Rusc2[label="1522 Rusc"]
+Fi2[label="1519 Fi"]
+Ald2[label="1534 Ald"]
+Greg[label="1525 Gref"]
+Zopp2[label="1524 Zopp"]
+Zopp3[label="1530 Zopp"]
+Pag2[label="1530-1 Pag"]
+Bind1[label="1531 Bind"]
+Bind2[label="1533 Bind"]
+Bind3[label="1543 Bind"]
+Bind4[label="1545 Bind"]
+Imp[label="1548 Imp"]
+Sess[label="1532 Sess"]
+Zio[label="1538 Zio"]
+Guad1[label="1539 Guad"]
+Guad2[label="1543 Guad"]
+Gio[label="1543 Gio"]
+Gio2[label="1549 Gio"]
+Sta[label="1531 Sta"]
+Na2[label="1507 Na, 1509Mi,1510,1512"] //VERY WEIRD, correct this
```

Comment:
I would write it like this:
Na2[label="\\[1507\\]Na,1509Mi,1510,1512"]
(Backslashes before opening and closing square brackets)

Reply:
Actually, I wonder if it is not more accurate as:

```gv
Na2[label="\[1507\] Na"] 
Mi1[label="1509 Mi"]
Mi2[label="1510 Mi"]
Mi3[label="1512 Mi"]

Na2->Mi1;
Mi1->Mi2;
Mi2->Mi3;
```
But it is not clear at all.

Reply:
The tree is a bit different as in Trovato (2014), Everything…

![image](https://user-images.githubusercontent.com/1204247/120498524-93286500-c3bf-11eb-8a7c-5d4f1f0ace56.png)

Reply:
> After the quarto editio princeps (1504) and the editions modelled on
> the first edition ( 1 5 0 7 , 1 5 0 9 , etc.), the new vulgate text is constituted by
> the Aldine edition of 1514. 

Ok, so it's

```gv

Na->Mi1;
Na->Mi2;
Na->Mi3;
```

Reply:
And no more weird arrow, I'm switching to this version.